### PR TITLE
implement click proxy for adzerk

### LINKED ISSUE #15619: Add proxy for Adzerk click URLs
Like #15618 but for click URLs.

### PR DESCRIPTION
**Pull request by eviljeff**
fixes #15619

### CODE REVIEW

File: docs/topics/api/shelves.rst [hunk 1]
```diff
@@ -117,3 +117,18 @@ The current implemenation forwards these impression pings to Adzerk.
...
     :form string impression_data: the signed data payload that was sent in the :ref:`sponsored shelf <sponsored-shelf>` response.
+
+
+---------------------------
+Sponsored Shelf Clicks
+---------------------------
+
+.. _sponsored-shelf-click:
+
+When and item on then sponsored shelf is clicked on by the user to navigate to the detail page this endpoint should be used to record the click.
```
Comment (bobsilverberg) [inline docs/topics/api/shelves.rst:128]: s/then sponsored/the sponsored

File: src/olympia/lib/settings_base.py [hunk 2]
```diff
@@ -1942,3 +1942,5 @@ def get_sentry_release():
 ADZERK_URL = f'https://e-{ADZERK_NETWORK_ID}.adzerk.net/api/v2'
 ADZERK_IMPRESSION_URL = f'https://e-{ADZERK_NETWORK_ID}.adzerk.net/i.gif?'
 ADZERK_IMPRESSION_TIMEOUT = 60  # seconds
+ADZERK_CLICK_URL = f'https://e-{ADZERK_NETWORK_ID}.adzerk.net/r?'
+ADZERK_CLICK_TIMEOUT = 60 * 60  # seconds
```
Comment (bobsilverberg) [inline src/olympia/lib/settings_base.py:1946]: Why is this 60 * 60, whereas `ADZERK_IMPRESSION_TIMEOUT` is just 60?

**Review by bobsilverberg — APPROVED** [review]
Looks good, thanks @eviljeff. Just a small comment and a question.

r+wc

File: docs/topics/api/shelves.rst (hunk 1 above)
Comment (bobsilverberg) [reply docs/topics/api/shelves.rst:128]: I just noticed there was a second error too, which you might be referencing. The inital "and" should be an "an".

Comment (eviljeff) [reply docs/topics/api/shelves.rst]: https://github.com/mozilla/addons-server/pull/15704/commits/8ef199ba322cc5554edeb055e4f92fee7e068854

File: src/olympia/lib/settings_base.py (hunk 2 above)
Comment (jvillalobos) [reply src/olympia/lib/settings_base.py:1946]: One hour sounds like more than enough. I'm good with it.

**Pull request merged**